src/main/java/com/denizenscript/depenizen/bukkit/properties/worldguard/WorldGuardNPCProperties.java

```diff
@@ -0,0 +1,99 @@
+package com.denizenscript.depenizen.bukkit.properties.worldguard;
+
+import com.denizenscript.denizen.objects.NPCTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.depenizen.bukkit.bridges.MythicMobsBridge;
+import com.denizenscript.depenizen.bukkit.objects.worldguard.WorldGuardRegionTag;
+import com.denizenscript.depenizen.bukkit.properties.mythicmobs.MythicMobsPlayerProperties;
+
+import net.citizensnpcs.trait.waypoint.WanderWaypointProvider;
+import net.citizensnpcs.trait.waypoint.Waypoints;
+import com.sk89q.worldguard.protection.regions.ProtectedRegion;
+
+public class WorldGuardNPCProperties implements Property {
```

Comment:
this whole class still needs to be replaced to the modern format for extension classes